### PR TITLE
added sideload endpoint

### PR DESCRIPTION
**Pull request by katarinarosiak**
Co-authored-by: Katarina Rosiak <katarinarosiak@gmail.com>
Co-authored-by: Tim Dronkers <tdronkers@gmail.com>
Co-authored-by: Scott Graham <scttgrhm7+public@gmail.com>
Co-authored-by: Miles Abbason <miles.abbason@gmail.com>

- created an endpoint for `GET` `/sideload`. The response contains a JSON file with data for `comparisonTypes`,  `regions`, `httpsMethods` and `assertionTypes`. 
- updated documentation for `GET /tests`, `GET tests/:id` and `GET /sideload` 
- changed the file exporting way for better naming of functions:  
Exporting: 
```javascript
module.exports.addNewTest = addNewTest;
module.exports.getTests = getTests;
module.exports.getTest = getTest;
module.exports.getSideload = getSideload;
```
Requireing: 
```javascript
const DB = require('../lib/db/query');
```
Using: 
```javascript
 await DB.addNewTest(ruleName, RuleArn, test);
```

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -74,3 +76,209 @@ Test my-new-test created
...
+    "updated_at": null
+  }
+]
+```
+
+#### 1.2.3 Forwarded Payload
+
+none
+
+## 1.3. GET /api/tests
```
Comment (tdronkers04) [inline docs/api.md]: I think this should be `/api/tests/:id`

File: docs/api.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 - [1. API Documentation](#1-api-documentation)
   - [1.1. POST /api/tests](#14-post-apitests)
+  - [1.2. GET /api/tests](#79-post-apitests)
+  - [1.3. GET /tests/:id](#138-post-apitests)
```
Comment (tdronkers04) [inline docs/api.md]: Add hyperlink to `[1.4. GET /tests/sideload]` ?

File: docs/api.md [hunk 3]
```diff
@@ -74,3 +76,209 @@ Test my-new-test created
...
+  },
+]
+
+```
+
+#### 1.3.3 Forwarded Payload
+
+none
+
+## 1.3. GET /api/sideload
```
Comment (tdronkers04) [inline docs/api.md]: restart numbering from `1.4` ...

**Review by tdronkers04 — CHANGES_REQUESTED** [review]
Great job Katarina, I just left a few minor  suggestions for the documentation

File: docs/api.md [hunk 4]
```diff
@@ -137,7 +138,7 @@ The scheduled tests are returned in JSON format with a 200 response status code.
 
 none
 
-## 1.3. GET /api/tests
+## 1.3. GET /api/test/:id
```
Comment (tdronkers04) [inline docs/api.md]: I think tests needs to be plural... `/api/tests/:id`

**Review by tdronkers04 — APPROVED** [review]

**Comment by katarinarosiak** [comment]
merging approved changes

**Pull request merged**